lib/matplotlib/axes/_axes.py

```diff
@@ -5268,62 +5268,69 @@ def _pcolorargs(funcname, *args, allmatch=False):
     def pcolor(self, *args, alpha=None, norm=None, cmap=None, vmin=None,
                vmax=None, **kwargs):
         """
-        Create a pseudocolor plot of a 2-D array.
+        Create a pseudocolor plot of a 2-D array using quadrilaterals.
```

Comment:
Not convinced "using quadrilaterals" really helps here (and yes I know how pcolor is implemented).

Reply:
I think we need some sort of addition to differentiate it from `imshow` and `matshow`. A better wording is welcome.

Reply:
Its explained below - I think the differentiation is useful.  OTOH to me the main difference w/ `imshow` is that the lateral and vertical spacing can be non-even (as opposed to distorted grids.)

Reply:
I could also do "Create a pseudocolor plot with a non-regular grid."

Reply:
I like that wording!